### PR TITLE
Localize text

### PR DESCRIPTION
**Pull request by jbydeley-d2l**

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+var gulp = require('gulp');
```
Comment (jwong-d2l) [inline gulpfile.js]: Is gulp just for merging the json file?

File: components/d2l-completion-status.html [hunk 2]
```diff
@@ -70,6 +72,11 @@
 				};
 			}
 
+			connectedCallback() {
+				super.connectedCallback();
+				this.loadResources(this.resolveUrl('../locales.json'));
+			}
+
 			_showComptionType( completion ) {
```
Comment (jwong-d2l) [inline components/d2l-completion-status.html]: comption lol... I'll fix that in another pr

File: components/d2l-completion-status.html [hunk 3]
```diff
@@ -48,7 +50,7 @@
 		@memberOf window.D2L.Polymer.Mixins;
 		@mixes D2L.Polymer.Mixins.CompletionStatusMixin
 		*/
-		class D2LCompletionStatus extends D2L.Polymer.Mixins.CompletionStatusMixin( Polymer.Element ) {
+		class D2LCompletionStatus extends Polymer.mixinBehaviors(D2L.PolymerBehaviors.LocalizeBehavior, D2L.Polymer.Mixins.CompletionStatusMixin( Polymer.Element )) {
```
Comment (tinglisD2L) [inline components/d2l-completion-status.html]: is this multiple inheritance?

Comment (jbydeley-d2l) [reply components/d2l-completion-status.html]: This is inheriting both Polymer 1/2 hybrid behaviors and Polymer 2 mixins.

File: gulpfile.js (hunk 1 above)
Comment (jbydeley-d2l) [reply gulpfile.js]: Yep. This was copy and pasted from the viewer since it does the same thing.

Comment (jwong-d2l) [reply gulpfile.js]: I think we should be able to get away without gulp, but that's for later us to discover.

Comment (RyanSweetD2L) [reply gulpfile.js]: I don't like gulp very much.  However, this is the easiest way for merging all the json files together.  The equivalent javascript is not fun.

**Pull request merged**